Makefile

```diff
@@ -96,8 +96,8 @@ endif
 #
 # Example:
 #   make check-test
-check-test: export KUBE_COVER= -cover -covermode=atomic
-check-test: export KUBE_RACE=  -race
+check-test: export COVERAGE_SPEC=-cover -covermode=atomic
+check-test: export DETECT_RACES=true
```

Comment:
@smarterclayton  I'm keeping backwards compatibility in `hack/test-go.sh` until we fully migrate everything, so we don't *need* to migrate these right now. However, these names are so heinous that if I were to have written them and made a pull with them @deads2k would come unplug my computer and throw it out of the window. They really need to go ...